dom.bs

```diff
@@ -1184,18 +1184,18 @@ for discussion).
 <h3 id=dispatching-events>Dispatching events</h3>
 
 <p>To <dfn export for=Event id=concept-event-dispatch>dispatch</dfn> an <var>event</var> to a
-<var>target</var>, with an optional <var>targetOverride</var>, run these steps:
+<var>target</var>, with an optional <var>legacy target override flag</var>, run these steps:
 
 <ol>
  <li><p>Set <var>event</var>'s <a>dispatch flag</a>.
 
  <li>
-  <p>If <var>targetOverride</var> is not given, let <var>targetOverride</var> be <var>target</var>.
+  <p>Let <var>targetOverride</var> be <var>target</var>, if <var>legacy target override flag</var>
```

Comment:
Nit: I think "**the** _legacy target override flag_" is more consistent with what we've done so far.